token-lending/cli/src/main.rs

```diff
@@ -311,7 +321,7 @@ fn main() {
         .get_matches();
 
     let mut wallet_manager = None;
-    let config = {
+    let mut config = {
```

Comment:
It doesn't look like this gets mutated, does it?

Reply:
yeh but i found that for testing I was needing to mutate it a lot (dryrun doesn't work for the init reserve because there are multiple txns so the later just always fail because the accounts aren't created so i had to have it run the first 2 for real then switch to dryrun)